Resources/Prototypes/Entities/Clothing/Hands/base_clothinghands.yml

```
@@ -57,6 +57,26 @@
     fiberMaterial: fibers-synthetic
   - type: FingerprintMask
 
+# needed so paws would be harmless and could squeak when hitting somebody
+- type: entity
+  parent: [ClothingHandsGlovesSyntheticBase, BaseSyndicateContraband]
+  id: ClothingHandsCatPawsBase
```

```suggestion
  id: ClothingHandsCatPawsBase
  categories: [ DoNotMap ]
```
Same as cat ears. Don't want mappers to get any ideas, these should be unique to the uplink.